tests/System/Routes/AddDonationRouteTest.php

```diff
@@ -283,4 +283,33 @@ private function newValidPayPalInput() {
 		];
 	}
 
+	public function testGivenValidCreditCardData_redirectsToIframeEmbeddingPage() {
```

Comment:
Shouldn't that be `testGivenValidCreditCardData_showsIframeEmbeddingPage`? Or do we really have a redirection here?